app.ts

```diff
@@ -0,0 +1,106 @@
+﻿/// <reference path="jquery.d.ts" />
+
+/**
+ * Michael Leibbrandt 
+ * Typescript Onboard Project
+ */
+
+namespace onboard {
+
+	// Variable declarations
+
+	let totRec = getMax(); // Total amount of records
+	let totHeadings;
+	let fromRecord = 0;
+	let toRecord = 15;
+	let count = 0;
+	let increase = 15;
+
+	$(document).ready(function () {
+
+		// Creation of table as well as insertion of data
+
+		function populate(to, from) {
+
+			// Retrieves columns or Headings from API GET call
+
+			$.getJSON("/columns", function (data) {
+				if (count === 0) {
+					$('#btn2').prop("disabled", true);
+				} else if (count === totRec) {
+					$('#btn1').prop("disabled", true);
+				}
+				// Retrieves and sets column names
+
+				let headers = "<tr>";
+				for (let i = 0; i < data.length; i++) {
+					headers += `<th>${data[i]}</th>`
+				}
+				totHeadings = data.length;
```

Comment:
If you move this declaration up before your `for` loop you can use the same variable (`totHeadings`) for instantiating the for-loop.